static/js/modules/streaks.js

```diff
@@ -0,0 +1,29 @@
+
+
+function generateStreakText(data)
+{
+
+    if (data['streak'] == 0) {
+        return "You currently don't have an active streak. Play the Prompt of the Day to start your streak!";
+    }
+    let first_blurbs = {
+        5: `You currently have a 🔥<b> ${data['streak']} day streak </b>🔥, keep it up!`,
+        10: `Good to see you again! You currently have a 🔥<b> ${data['streak']} day streak </b>🔥, impressive!`,
+        25: `Woah, 🔥<b> ${data['streak']} day streak </b>🔥 and going strong!!`,
+    };
+
+    let disp = `Didn't think you'd make it this far, but you did! 🔥<b> ${data['streak']} day streak </b>🔥!!`
+    for (const threshold in first_blurbs) {
```

Comment:
for...in... loops don't maintain order when iterating over a hashtable in JS. Should use an array of objects or if...else... statements instead https://stackoverflow.com/questions/280713/elements-order-in-a-for-in-loop

Reply:
Oh shit I had no idea, I will change that